contracts/KetlAttestation.sol

```diff
@@ -79,24 +79,30 @@ contract KetlAttestation is
   // Attestations
   uint32 public currentTokenId;
   uint public attestorPublicKey;
-  mapping(uint => uint) public attestationMerkleRoots;
+  mapping(uint attestationType => uint merkleRoot)
```

Comment:
It would be good to have a reference for AttestationType. I found out that we can use `enum`s in Solidity, here's the example:
```sol
    enum Rarity {
        original, // 0
        rare, // 1
        super_rare // 2
    }
    
    Rarity public rarity; 
```
Maybe we can use a enum here too?

Reply:
I don't think we want to use an enum in the contract because we would need to upgrade the contract every single time we add a new attestationType.